prisma-fmt/src/code_actions/multi_schema.rs

```diff
@@ -142,13 +142,12 @@ pub(super) fn add_schema_to_schemas(
                 formatted_attribute,
                 true,
                 // todo: update spans so that we can just append to the end of the _inside_ of the array. Instead of needing to re-append the `]` or taking the span end -1
-                Span::new(span.start, span.end - 1),
+                Span::new(span.start, span.end - 1, psl::parser_database::FileId::ZERO),
                 params,
             )
         }
         None => {
-            let has_properties = datasource.provider_defined()
-                || datasource.url_defined()
+            let has_properties = datasource.provider_defined() | datasource.url_defined()
```

Comment:
Why the binary `|`?

Reply:
I did not notice it. My bet it's just a typo, we should revert back to `||`